src/controller/ReadInteraction.h

```diff
@@ -196,14 +196,14 @@ struct ReportEventParams : public app::ReadPrepareParams
 
 template <typename DecodableEventType>
 CHIP_ERROR ReportEvent(Messaging::ExchangeManager * apExchangeMgr, EndpointId endpointId,
-                       ReportEventParams<DecodableEventType> && readParams)
+                       ReportEventParams<DecodableEventType> && readParams, bool aIsUrgentEvent = false)
```

Comment:
This does not need defaulting: the callers are all in this file.  Just add the new arg as a required arg.

Reply:
updated. thanks